code/modules/antagonists/blob/overmind.dm

```diff
@@ -290,8 +290,9 @@ GLOBAL_LIST_EMPTY(blob_nodes)
 		forceMove(NewLoc)
 		return TRUE
 
-/mob/camera/blob/mind_initialize()
+/mob/camera/blob/mind_initialize(skip_initialization=FALSE)
 	. = ..()
-	var/datum/antagonist/blob/B = mind.has_antag_datum(/datum/antagonist/blob)
-	if(!B)
+	if(!skip_initialization && !mind.has_antag_datum(/datum/antagonist/blob))
 		mind.add_antag_datum(/datum/antagonist/blob)
+
+	announce_non_antag_mind_policy(src)
```

Comment:
The only case this will happen is due to a bug. If someone is inside of a blob overmind, they should be antagonising

Reply:
I will hardly happen but what if your crew mind is somehow swapped with blob...?

Reply:
I think we should ditch the non antag mind policy for this PR since it needs more specific considerations. For example there are cases where you could be an antagonist that would have this message still display